janusgraph-hadoop/src/test/resources/org/janusgraph/hadoop/formats/graphson/incremental-custom-cerberus-load.groovy

```diff
@@ -21,7 +24,7 @@ JanusGraphVertex getOrCreateVertex(faunusVertex, graph, context, log) {
     if (null == uniqueValue) {
         throw new RuntimeException("The provided Faunus vertex does not have a property for the unique key: " + faunusVertex)
     }
-    Iterator<Vertex> itty = graph.query().has(uniqueKey, uniqueValue).vertices().iterator()
+    Iterator<Vertex> itty = graph.query().has(uniqueKey, uniqueValue).vertiexStream().iterator()
```

Comment:
Did you test this? I don't know the method call, but it looks to me like this should be `vertexStream()` not `vertiexStream()`.